airship_tempest_plugin/tests/api/shipyard/test_document_staging.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2018 AT&T Corp
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+import logging
+
+from airship_tempest_plugin.tests.api.shipyard import base
+
+from tempest import config
+from tempest.lib import decorators
+from tempest.lib import exceptions
+
+CONF = config.CONF
+LOG = logging.getLogger(__name__)
+
+
+class DocumentStagingTest(base.BaseShipyardTest):
+    def _get_collection_id(self):
+        resp = self.shipyard_document_staging_client.get_configdocs_status()
+        self.assertTrue(len(resp[0]) > 0,
+                        'No configdocs available, nothing to test')
+        return resp[0]['collection_name']
+
+    @decorators.idempotent_id('db351ee5-a608-4492-b705-6e6c654a60e7')
+    def test_get_collection_list(self):
```

Comment:
does not seem like a good name for this test as its getting "config docs status"